tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,199 @@
+import logging
+import hail as hl
+import hail.expr.aggregators as agg
+
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+from os.path import dirname
```

Comment:
```suggestion
import argparse
import logging
import matplotlib.pyplot as plt
import pandas as pd
from os.path import dirname

import hail as hl
import hail.expr.aggregators as agg

from gnomad.utils.reference_genome import get_reference_genome
```
we try to order the imports alphabetically on our team. here's a link to the Python style guide for more reference about recommended practices around imports/spacing between imports: https://peps.python.org/pep-0008/#imports